src/js/firstDiscoveryEditor.js

```diff
@@ -37,11 +37,66 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                         }
                     }
                 }
+            },
+            navButtons: {
+                type: "gpii.firstDiscovery.navButtons",
+                container: "{that}.dom.navButtons",
+                createOnEvent: "onCreateNavButtons",
+                options: {
+                    gradeNames: ["fluid.prefs.msgLookup"],
+                    members: {
+                        messageResolver: "{firstDiscoveryEditor}.msgResolver"
+                    },
+                    strings: {
+                        back: "{that}.msgLookup.back",
+                        next: "{that}.msgLookup.next",
+                        start: "{that}.msgLookup.start",
+                        finish: "{that}.msgLookup.finish"
+                    },
+                    panelTotalNum: "{firstDiscoveryEditor}.panelTotal",
+                    model: {
+                        currentPanelNum: "{firstDiscoveryEditor}.model.currentPanelNum"
+                    }
+                }
             }
         },
+        modelListeners: {
+            "currentPanelNum": "{that}.showPanel"
+        },
         events: {
-            onPrefsEditorReady: null
+            onPrefsEditorReady: null,
+            onCreateNavButtons: null
+        },
+        listeners: {
+            "onPrefsEditorReady.getPanelTotal": {
+                listener: "gpii.firstDiscovery.getPanelTotal",
+                args: ["{that}"],
+                priority: "first"
+            },
+            "onPrefsEditorReady.showInitialPanel": {
+                listener: "{that}.applier.change",
+                args: ["currentPanelNum", 1]
+            },
```

Comment:
You should be able to prevent the model listener from triggering on init. Then you can call the method you use for handling it, when the time is right.